src/main/java/org/kairosdb/datastore/cassandra/ClusterConnection.java

```diff
@@ -313,7 +310,7 @@ public ClusterConnection(CassandraConfiguration cassandraConfig, CassandraClient
 		m_cassandraClient = cassandraClient;
 		m_clusterType = clusterType;
 
-		m_alwaysUseTagIndexedLookup = tagIndexMetricNames.equals(ImmutableMultimap.of("*", "*"));
```

Comment:
Now that the config format is an object, not an array, we need to check for quoted *s. Unquoted *s aren't valid config object keys.